### PR TITLE
Moban test requirements

### PR DESCRIPTION
**Pull request by jayvdb**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by jayvdb** [comment]
This depends on https://github.com/coala/coala/pull/5506

**Comment by jayvdb** [comment]
See also https://github.com/coala/coala-bears/pull/2528

**Comment by MacBox7** [comment]
All PR have similar changes. I guess merging this first would make others redundant.

File: test-requirements.txt [hunk 1]
```diff
@@ -6,15 +6,15 @@ codecov~=2.0.5
...
+pytest-cov~=2.4
 pytest-env~=0.6.0
 pytest-error-for-skips~=1.0
 pytest-instafail~=0.3.0
 pytest-mock~=1.1
 pytest-reorder~=0.1.0
 pytest-reqs~=0.0.6
-pytest-timeout~=1.0
-pytest-xdist~=1.14
+pytest-timeout~=1.2
```
Comment (sangamcse) [inline test-requirements.txt]: This needs update.

Comment (sangamcse) [reply test-requirements.txt]: Rebase required for this change.

**Comment by jayvdb** [comment]
Rebased.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5484/commits/edd7be67c0183532aacdaa609fb5afee3f37a8cb https://github.com/coala/coala/pull/5484/commits/5b8616cfca5aafc142a98d03fcd8c9be3c5993dd

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**